src/app/components/main/main.html

```diff
@@ -1,32 +1,34 @@
-
+<head>
+	<link rel="stylesheet" href="./main.css">
+</head>
+<body>
 	<div class="container">
-		<div class="row">
-			<div class="appname"><h4>APP_NAME</h4></div>
+			<h4>APP_NAME</h4>
 			<div class="menu">
 
-				<div class="userinfo"><img src="img/default-user-image.png">Name Surname<div class="moreinfo"><button><img src="img/dropdownbtn.png"></button></div></div>
+				<div class="userinfo"><img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRpWMOVHeFzDQhg3e7gzqgJLdRYQ44EgxKM5B2XMNSDhxptbN7AE-re6aE"><span>Name Surname</span><div class="moreinfo"><button><img src="dropdownbtn.png"></button></div></div>
```

Comment:
is there any purpose to change to encrypted image. by referring to third party site you will make you app dependent to it. If that website won't work you will get no result in you app. I will suggest to put the simple url to you image, unless you can get it from json.